packages/components/src/components/post-header/post-header.tsx

```diff
@@ -273,6 +248,20 @@ export class PostHeader {
     );
   }
 
+  componentWillRender() {
+    window.addEventListener('resize', this.throttledResize, { passive: true });
+    window.addEventListener('scroll', this.handleScrollEvent.bind(this), {
+      passive: true,
+    });
+    this.scrollParent.addEventListener('scroll', this.handleScrollEvent.bind(this), {
+      passive: true,
+    });
```

Comment:
Why do we need both listeners?

Reply:
Because if the scrollParent is the <html/> element, the this.scrollParent listener is not working and vice versa.